_posts/2022-04-22-security-debrief.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: 'Security Debrief: April 17-23, 2022'
+author_avatar: https://avatars.githubusercontent.com/u/2421172?v=4
+author_name: Alex Smith
+author_github_username: tradesmanhelix
+excerpt: A look back at the top security stories from the week.
+---
+
+# This Week in Cybersecurity: April 17-23, 2022
```

Comment:
@tradesmanhelix , it might be great to add a short blurb introducing yourself saying how you recently joined the Maxwell Engineering team and one of your interests is cybersecurity. And that going forward, you plan to collect and share security stories from the week on the Maxwell Engieering Blog.